apps/src/p5lab/AnimationPicker/AnimationPickerBody.jsx

```diff
@@ -272,6 +272,12 @@ export default class AnimationPickerBody extends React.Component {
                   onUploadClick={onUploadClick}
                   shouldWarnOnAnimationUpload={shouldWarnOnAnimationUpload}
                   isBackgroundsTab={isBackgroundsTab}
+                  inRestrictedShareMode={false}
+                  teacherHasConfirmedUploadWarning={false}
+                  refreshInRestrictedShareMode={() => {}}
+                  refreshTeacherHasConfirmedUploadWarning={() => {}}
+                  showingUploadWarning={() => {}}
+                  exitedUploadWarning={() => {}}
```

Comment:
Rather than providing all these props, should we be wrapping this page in a `<Provider>`? I don't have too much context on this button, but I'm assuming we'd need some of this functionality to work, right? Or if not, should this button even be visible on this page?

Reply:
I'm going to go ahead and update PR back to draft. Thanks!